README.md

```diff
@@ -62,6 +62,15 @@ $ openstack tcib container image build --config-file containers.yaml --repo-dir
 
 The `--tcib-extras tcib_package=` is required so that non-template files are sourced from the local `container-images/kolla` instead of from the `python-tcib-containers` package in the container build.
 
+## Matching container images to tcib commits
+
+1. Look at the container image in quay.io
+   (e.g. https://quay.io/repository/podified-antelope-centos9/openstack-tempest-extras?tab=tags) and take the tag
+2. The tag is the commit hash in https://trunk.rdoproject.org/centos9-antelope/current-podified/
+   (e.g. https://trunk.rdoproject.org/centos9-antelope/current-podified/8f/cc/8fcc848d6c766b48142f0ffef9e34937/)
+3. Look into the `delorean.repo` file and search for tcib, the file will point to the tcib repo hash
```

Comment:
it should be search for podified instead of search for tcib as the repo contains multiple packages and repo name will be of the last package build from that component

Reply:
I don't get it. The delorean.repo file has no data about the tempest-extra image only data about the tcib repo.
// later
I think I see it. So there is `[delorean-component-podified]`  in the delorean.repo that in my examples always points to tcib. I guess if I start my search from a container image tag from quay.io then I always end up with a tcib reference at `[delorean-component-podified]` . Isn't it?

Reply:
so if you see in https://trunk.rdoproject.org/centos9-antelope/component/podified/current-podified/ there are 3 rpms:-
ansible-config_template-2.1.1-0.20231113175736.04e621b
edpm-image-builder-0.0.1-0.20231108021806.6b6bcc0
python-tcib-0.2.1-0.20231214102319.7bf4dba

Since other two are not build over a month, you see always tcib there, but as soon as there will be new build, name will point to that rpm, basically name contains the rpm last build for the component

Reply:
for reference an old repo where other rpm name exist https://trunk.rdoproject.org/centos9-antelope/current-podified/78/75/7875d401b14a7e53d94789026df40798/delorean.repo

Reply:
ack